specification/chaos/Chaos.Management/client.tsp

```diff
@@ -33,27 +33,27 @@ interface Experiments {
   executionDetails is Microsoft.Chaos.ExperimentExecutions.getExecutionDetails;
 }
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
 @armResourceOperations
 interface Capabilities extends Microsoft.Chaos.Capabilities {}
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
 @armResourceOperations
 interface CapabilityTypes extends Microsoft.Chaos.CapabilityTypes {}
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
 @armResourceOperations
 interface Operations extends Microsoft.Chaos.Operations {}
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
 @armResourceOperations
 interface OperationStatuses extends Microsoft.Chaos.OperationStatuses {}
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
 @armResourceOperations
 interface Targets extends Microsoft.Chaos.Targets {}
 
-@operationGroup("java,go")
+@operationGroup("java,go,javascript")
```

Comment:
there is un-necessary breaking and pls use clientname to keep it, that is for operation listall renamed as list.

![image](https://github.com/user-attachments/assets/9ccb645b-0b50-439d-9ff3-6717d083dbbc)